.changeset/chilly-kids-arrive.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/blocks-admin": minor
```

Comment:
Is removing deprecated functionality a minor change? Sounds more like a major change (but doesn't matter so much as all the changes will be released in v5.0.0 anyway)

Reply:
You're right, removing deprecated code is a major change.